### PR TITLE
Code For America

### PR DESCRIPTION
**Pull request by dirkkelly**
:us:

### CODE REVIEW

**Comment by dirkkelly** [comment]
:bowtie: ready for review

File: lib/digest_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    def today_as_digest_day
+      Date.today.strftime('%A').downcase
+    end
+
+    def digest_day?
+      digest_day == today_as_digest_day
+    end
+
+    def send_notifications
+      ::Subscription.notifiables.where(channel: 'email').paged_each do |subscription|
```
Comment (excepttheweasel) [inline lib/digest_helper.rb]: This should be `Subscription.email`

**Comment by excepttheweasel** [comment]
:+1:

File: lib/digest_helper.rb (hunk 1 above)
Comment (dirkkelly) [inline lib/digest_helper.rb]: Agreed, let's make another pull request onto `codeforamerica/master`
